service/create/error.go

```diff
@@ -10,3 +10,10 @@ var invalidConfigError = errgo.New("invalid config")
 func IsInvalidConfig(err error) bool {
 	return errgo.Cause(err) == invalidConfigError
 }
+
+var keyPairNotFoundError = errgo.New("key pair not found")
+
+// IsKeyPairNotFound asserts keyPairNotFoundError.
+func IsKeyPairNotFound(err error) bool {
+	return errgo.Cause(err) == keyPairNotFoundError
+}
```

Comment:
This is no longer necessary I guess. At least it is not used in the code anymore.